README.md

```diff
@@ -97,7 +97,7 @@ configurations.
 ### Misc Targets
 * `make clean`: Delete binaries built from source
 =======
-* `make run`: Runs the broker with the default profile, configured via `/etc/broker-config.yaml`
+* `make run`: Runs the broker with the default profile, configured via `/etc/config.yaml`
```

Comment:
I'm not sure what makes sense here:
I see in the new prep and run scripts:
`BROKER_CONFIG=${GENERATED_BROKER_CONFIG}` and
`GENERATED_BROKER_CONFIG="../etc/generated_local_development.yaml"`